azkaban-common/src/main/java/azkaban/utils/AbstractMailer.java

```diff
@@ -53,24 +48,6 @@ public AbstractMailer(final Props props) {
 
     this.mailSender = props.getString("mail.sender", "");
     this.usesAuth = props.getBoolean("mail.useAuth", true);
-
-    this.clientHostname = props.get("server.hostname");
-    this.clientPort = props.getInt("server.port");
-    this.usesSSL = props.getBoolean("server.useSSL");
-
-    if (this.usesSSL) {
-      this.referenceURL =
-          "https://" + this.clientHostname
-              + (this.clientPort == 443 ? "/" : ":" + this.clientPort + "/");
-    } else {
-      this.referenceURL =
-          "http://" + this.clientHostname
-              + (this.clientPort == 80 ? "/" : ":" + this.clientPort + "/");
-    }
-  }
-
-  public String getReferenceURL() {
```

Comment:
1). If me, I would not clean up this method, though Intellij can not identify its caller.
2). There are quite a few methods below like `getMailHost`, which are also never called up. I would clean them together in future.